keras/src/losses/losses.py

```diff
@@ -99,7 +99,7 @@ class MeanAbsoluteError(LossFunctionWrapper):
 
     def __init__(
         self,
-        reduction="sum_over_batch_size",
+        reduction="mean_absolute_error",
```

Comment:
?

Reply:
@fchollet 
The reduction function for `keras.metrics.MeanAbsoluteError` is `mean_absolute_error` and the reduction function for `keras.losses.MeanAbsoluteError` is `sum_over_batch_size`.
The user here https://github.com/keras-team/keras/issues/19740 faces an issue when using sample weights for her training. the loss computes a different value from the associated metric during training.
More details here - https://github.com/keras-team/keras/issues/19740

Reply:
i just realized `mean_absolute_error` is not one of the options, I can add that.